ttnn/visualizer/__init__.py

```diff
@@ -641,13 +639,31 @@ def plot_tensor(tensor):
 
         return components(plot)
 
+    def get_tensor_attributes(tensor_record):
+        output = "<table>"
+        output += f"""
+            <tr><th>Shape</th><th>{tensor_record.shape}</th></tr>
+            <tr><th>Dtype<t/h><th>{tensor_record.dtype}</th></tr>
+            <tr><th>Layout</th><th>{tensor_record.layout}</th></tr>
+        """
+        if tensor_record.memory_config is not None:
+            tensor_memory_config = tensor_record.memory_config.replace("(", "(<br>").replace(",", ",<br>")
+            output += f"""
+                <tr><th>Device<t/h><th>{tensor_record.device_id}</th></tr>
+                <tr><th>Memory Config</th><th>{tensor_memory_config}</th></tr>
+            """
+        output += "</table>"
+        return output
```

Comment:
Are we going to fix this later?   It seems like injecting html into html is a bit of a hack.

Reply:
Yes, we should fix it. I do this in a bunch of places. All of bokeh plots are html being injected into html